decentraland/components/presentational/BuyLand.test.js

```diff
@@ -25,8 +26,28 @@ describe("BuyLand", () => {
       ).toMatchSnapshot();
     });
 
+    it("waiting for setup - waiting for account generation", async () => {
+      const accountInfo = {
+        account: null,
+        setupInProgress: true,
+        fundedWithEthers: false,
+        fundedWithMana: false,
+        approvedMarketplace: false,
+      };
+      expect(
+        shallow(
+          <BuyLand
+            landForSale={parcelForSale}
+            onBuy={onBuy}
+            accountInfo={accountInfo}
```

Comment:
Does `BuyLand` still need `accountInfo` in this format? Should we move it over to using the enum state from redux via the connect state to props in the screen that renders this presentational component?

Reply:
Yes, I'll do that.

Reply:
Okay, sounds good!

Reply:
Done.

Reply:
Thanks! Resolving